packages/react-email/source/commands/build.ts

```diff
@@ -0,0 +1,32 @@
+import { glob } from 'glob';
+import esbuild from 'esbuild';
+import { render } from '@react-email/render';
+import { unlinkSync, writeFileSync } from 'fs';
+
+/*
+  This first builds all the templates using esbuild and then puts the output in the `.js`
+  files. Then these `.js` files are imported dynamically and rendered to `.html` files
+  using the `render` function.
+ */
+export const build = async (outDir: string, pretty: boolean) => {
+  const allTemplates = glob.sync('emails/*.tsx');
```

Comment:
Can you add support for `.jsx` as well?

Context:
https://github.com/zenorocha/react-email/issues/231
https://github.com/zenorocha/react-email/pull/236